Check i2c

jonl40 opened the pull request.

buzuloiu committed:
yost init

jonl40 committed:
commit

buzuloiu commented on test-i2c/test-i2c.ino:
```diff
@@ -1,4 +1,4 @@
-//imports
+ //imports
```
this space shouldn't be here

jonl40 committed:
Add files via upload

imu locks up.

jonl40 commented:
imu locks up. Also, i'm not to sure where to put set axis with tare, and software reset.

jonl40 committed:
wip

jonl40 committed:
Add files via upload

work in progress.
read_euler() and read_accel() work. 
axis_w_tare();// Tare with current orientation seems to lock up imu and sensors just read 0 for read_euler(), read_accel(), and read_imu().

buzuloiu commented on test-i2c/Yost.h:
```diff
@@ -0,0 +1,85 @@
+     float eulerFloat;
+    } euler;
+
+   //float angle_euler[3]; // store each float (every 4 bytes) into an index of the xyz_euler array
```
Should this be commented out?

jonl40 replied:
It should be commented out. I'll remove it and make a new pull request

buzuloiu commented on test-i2c/IMU_v2.ino:
```diff
@@ -0,0 +1,141 @@
+#include <Wire.h>
+
+#define TRUE 1
+#define FALSE 0
```
I don't think you need to define these in Arduino, I think `true` and `false` are already mapped to `1` and `0`

buzuloiu commented on IMU_v2/IMU_v2.ino:
```diff
@@ -0,0 +1,134 @@
+//Component: Yost 3-Space™ LX Embedded part number TSS-LX
```
- I think you can delete this whole folder if the `test-i2c` folder is the one you want to use

buzuloiu commented on test-i2c/test-i2c.ino:
```diff
@@ -0,0 +1,144 @@
+ //imports
```
- you should copy the contents of `IMU_V2.ino` into this file and delete `IMU_V2.ino`,  `.ino` files should be in a folder with the same name as them

buzuloiu reviewed (commented):
This looks good, just needs some cleaning up

jonl40 committed:
imu sensor data in struct now

jonl40 committed:
merge
